deps/oblib/src/lib/oblog/ob_log.cpp

```diff
@@ -134,7 +134,16 @@ inline void ob_log_unlink(const char *file_cstr)
 {
   unlink(file_cstr);
   ObString file_name(file_cstr);
-  unlink(ObLogCompressor::get_compression_file_name(file_name).ptr());
+  ObString::obstr_size_t  buf_size = file_name.length() + 1 + sizeof(ObString);
```

Comment:
why plus sizeof(ObString), not the ".zst" suffix length？

Reply:
I write 4,but 4 is not clear, 
so the suffix length of  ".zst" is clear.